README.md

```diff
@@ -8,24 +8,19 @@ triggered by GitHub pushes or changes in the Contentful CMS it uses.
 This project can be extended for other webhook processing.
 
 # Usage
-This project requires Python 3.7+ and [Nix][nix].
+This project requires Python 3.12.
```

Comment:
Maybe mention that `uv` can install a local version of the correct python using `uv sync`.